### PR TITLE
Collections: Grouping and opening content

### PR DESCRIPTION
**Pull request by mikeger**
# Changes

- Added "Library" icon and updated stylekit
- Added section headers and made them interactive
- Added ability to show only certain section in the controller

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/Collections/CollectionsViewController.swift [hunk 1]
```diff
@@ -19,11 +19,37 @@
...
+        self = type(of: self).allAsArray[Int(index)]
+    }
+    
+    public static let none = CollectionsSectionSet(rawValue: 0)
+    public static let images = CollectionsSectionSet(rawValue: 1)
+    public static let filesAndAudio = CollectionsSectionSet(rawValue: 1 << 1)
+    public static let videos = CollectionsSectionSet(rawValue: 1 << 2)
+    public static let links = CollectionsSectionSet(rawValue: 1 << 3)
+    
+    public static let all = CollectionsSectionSet(rawValue: UInt.max)
```
Comment (daehn) [inline Wire-iOS/Sources/UserInterface/Collections/CollectionsViewController.swift]: I think `all` should be expressed as an union of all the other types (see https://developer.apple.com/reference/swift/optionset):
```swift
public static let all: CollectionSectionSet = [images, filesAndAudio, videos, links] 
```

File: Wire-iOS/Sources/UserInterface/Conversation/ConversationViewController.m [hunk 2]
```diff
@@ -421,7 +403,7 @@ - (void)setConversation:(ZMConversation *)conversation
 
 - (void)setupNavigatiomItem
 {
-    self.titleView = [[ConversationTitleView alloc] initWithConversation:self.conversation];
+    self.titleView = [[ConversationTitleView alloc] initWithConversation:self.conversation interactive:YES];
```
Comment (daehn) [inline Wire-iOS/Sources/UserInterface/Conversation/ConversationViewController.m]: The `titleView` should only be interactive if the conversation's connection is not an outgoing connection request, see https://github.com/wireapp/wire-ios/pull/491/files#diff-64b0ad5fc5c36accc32ddffdef3c09ccL50

Comment (mikeger) [inline Wire-iOS/Sources/UserInterface/Conversation/ConversationViewController.m]: Right, but this logic is not changed I think, or I am missing something?

**Comment by mikeger** [comment]
@daehn updated!

**Pull request merged**